File: VonexDealer.Vue/ClientApp/models/Dealer.js
export class Dealer {
    constructor({ dealerID = null,
        dealerFullName = null,
        dealerEmail = null,
        uBDealerID = null } = {}) {
        this.dealerID = dealerID;
        this.dealerFullName = dealerFullName;
        this.dealerEmail = dealerEmail;
        this.uBDealerID = uBDealerID;
    }
}

export function createDealer(data) {
    return Object.freeze(new Dealer(data));
}

File: VonexDealer.API/Models/Order/Dealer.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace VonexDealer.API.Models.Order
{
    /// <summary>
    /// Existing Dealer information taken from Utilibill and OMS system.
    /// </summary>
    public class Dealer
    {
        public Int64 DealerID { get; set; }
        public Int64 UBDealerID { get; set; }
        [Required]
        public string DealerFullName { get; set; }
        [Required]
        public string DealerEmail { get; set; }

    }
}
